gcovr/tests/simple1/Makefile

```diff
@@ -18,10 +18,16 @@ run: txt xml html sonarqube json json_summary coveralls
 txt:
 	./testcase
 	# a couple of tests about failure thresholds
-	$(GCOVR) --fail-under-line 80.1; test $$? -eq 2
-	$(GCOVR) --fail-under-branch 50.1; test $$? -eq 4
-	$(GCOVR) --fail-under-line 80.1 --fail-under-branch 50.1; test $$? -eq 6
-	$(GCOVR) --fail-under-line $(FAIL_UNDER_LINE) --fail-under-branch 49.9
+	$(GCOVR) --fail-under-line 80.1 2>fail_under.stderr; test $$? -eq 2 || ( cat fail_under.stderr & exit 1 )
+	grep -F "failed minimum line coverage" fail_under.stderr
+	$(GCOVR) --fail-under-branch 50.1 2>fail_under.stderr; test $$? -eq 4 || ( cat fail_under.stderr & exit 1 )
+	grep -F "failed minimum branch coverage" fail_under.stderr
+	$(GCOVR) --fail-under-line 80.1 --fail-under-branch 50.1 2>fail_under.stderr; test $$? -eq 6 || ( cat fail_under.stderr & exit 1 )
+	grep -F "failed minimum line coverage" fail_under.stderr
+	grep -F "failed minimum branch coverage" fail_under.stderr
+	$(GCOVR) --fail-under-line $(FAIL_UNDER_LINE) --fail-under-branch 49.9 2>fail_under.stderr; test $$? -eq 0 || ( cat fail_under.stderr & exit 1 )
+	grep -F "failed minimum line coverage" fail_under.stderr; test $$? -eq 1
+	grep -F "failed minimum branch coverage" fail_under.stderr; test $$? -eq 1
```

Comment:
I think this can be simplified to `! grep ...`?

Reply:
But then everithing not 0 will success, from `grep` manpage:
```
EXIT STATUS
     The grep utility exits with one of the following values:

     0     One or more lines were selected.
     1     No lines were selected.
     >1    An error occurred.
```